runtime/frequency/src/lib.rs

```diff
@@ -150,6 +148,31 @@ impl Contains<RuntimeCall> for BaseCallFilter {
 	}
 }
 
+impl BaseCallFilter {
+	fn is_utility_call_allowed(call: &RuntimeCall) -> bool {
+		match call {
+			RuntimeCall::Utility(pallet_utility::Call::batch { calls, .. }) |
+			RuntimeCall::Utility(pallet_utility::Call::batch_all { calls, .. }) |
+			RuntimeCall::Utility(pallet_utility::Call::force_batch { calls, .. }) =>
+				!calls.iter().any(Self::is_blocked_call),
```

Comment:
this O(n) for now, thinking of ways to optimize this, given this is in wasm

Reply:
This is  very rudimentary optimization https://github.com/LibertyDSNP/frequency/compare/main..599_optimized_for_wasm

but we can move out to a const BTreeMap to maintain block calls